CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ New:
 - Create `mosaic-animation` library, that provides various possibilities for animating Mosaic. An analog of [androidx.compose.animation-core](https://developer.android.com/reference/kotlin/androidx/compose/animation/core/package-summary).
 - Add `IntrinsicSize` and related `Modifier.width/height/requiredWidth/requiredHeight`.
 - New `mosaic-testing` artifact for testing Mosaic.
+- Add `Modifier.composed`.
```

Comment:
Please add at least a full sentence here about what it's for. Can be copied from the docs or something.